configure.ac

```diff
@@ -137,6 +137,13 @@ fi
 AS_IF([test "x$with_Bsymbolic" = "xyes"], [JSON_BSYMBOLIC_LDFLAGS=-Wl[,]-Bsymbolic-functions])
 AC_SUBST(JSON_BSYMBOLIC_LDFLAGS)
 
+# Enable symbol versioning on GNU/Linux
+JSON_SYMVER_LDFLAGS=
+AC_CANONICAL_HOST
+AS_CASE([$host],
```

Comment:
can you please check for `$host_os` instead? this way it is easier to match

Reply:
Sure.

Reply:
Actually, based on the check you suggested for glibc on CMake, I might as well mirror it directly and use `AC_CHECK_DECL` - it seems like that will maximize the chance that they give the same answer.